trimmer_decorator

```diff
@@ -0,0 +1,5 @@
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0, 10]
```

Comment:
- [ ] Please make sure to write the correct method for the correct name in the trimmer class.
The name has a maximum of 10 characters. If it's longer it should trim the word.